patches/0002-Add-crypto-backend-foundation.patch

```diff
@@ -813,51 +813,60 @@ index 00000000000000..b0cdd44d81c753
 +	panic("boringcrypto: not available")
 +}
 diff --git a/src/crypto/hkdf/hkdf.go b/src/crypto/hkdf/hkdf.go
-index 7cfbe2c60de356..78139ed6170da5 100644
+index 7cfbe2c60de356..925b839b73cb0c 100644
 --- a/src/crypto/hkdf/hkdf.go
 +++ b/src/crypto/hkdf/hkdf.go
-@@ -11,6 +11,7 @@
+@@ -11,8 +11,9 @@
  package hkdf
  
  import (
+-	"crypto/internal/fips140/hkdf"
```

Comment:
You probably have an out of dated version of `main`. Please rebase because `crypto/hkdf` was already patched in https://github.com/microsoft/go/pull/1449.

Reply:
https://github.com/microsoft/go/blob/646086032c418e016eb6e368eee9492ce7ddd083/patches/0002-Add-crypto-backend-foundation.patch#L824

main has exact same version

Reply:
Fixed